### DISCUSSION TITLE
All BentoJS components not working at the moment

### DISCUSSION THREAD

**Discussion by justinjadeee** [ampproject/bento#28] [General]
I can't use any BentoJS components as of the moment and as checking the console tab, it shows a lot of error on bento js components. 

Is there any current updates that is happening or is there any issues happening at the moment?

**Comment by erwinmombay** [comment]
@justinjadeee we're looking into this.

**Comment by erwinmombay** [comment]
@justinjadeee we just pushed out a fix. would you mind confirming if things look good on your end?

**Reply by justinjadeee** [reply]
Hi @erwinmombay, thanks for looking into this. Everything seems to be working now, appreciate your help!
